src/librustc/lint.rs

```diff
@@ -1,30 +1,15 @@
 use std::cmp;
 
 use crate::ich::StableHashingContext;
-use crate::lint::context::CheckLintNameResult;
```

Comment:
Seems like git lost the rename here -- maybe we can split the raw move into a separate commit so it keeps track of the rename and we don't lose history?

Reply:
(We reached the conclusion in private chat that GitHub is being buggy here and that blame is in fact intact.)